recipe/meta.yaml

```diff
@@ -14,11 +14,12 @@ source:
 build:
   number: 0
   noarch: python
+  skip: True  # [py<36]
   script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   host:
-    - python >=3.6
+    - python
```

Comment:
Do we need to keep the `>=3.6` in here? It is in the setup.py:
https://github.com/marshmallow-code/marshmallow-sqlalchemy/blob/0.26.1/setup.py#L48

I know we don't package anything less anymore.

Reply:
Yeah, there should be `python` without `>=3.6`